bngblaster install fails when building on 26.04 LTS

**Describe the bug**

Followed the installation process in ubuntu 26.04 LTS release and got this error while trying to make install 
`bngblaster/src/isis/isis_utils.c:114:9: error: assignment discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
  114 |     ptr = strchr(str, '/');
      |         ^
cc1: all warnings being treated as errors
make[2]: *** [code/bngblaster/CMakeFiles/bngblaster.dir/build.make:933: code/bngblaster/CMakeFiles/bngblaster.dir/src/isis/isis_utils.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:202: code/bngblaster/CMakeFiles/bngblaster.dir/all] Error 2
make: *** [Makefile:156: all] Error 2`

Version (`bngblaster -v`):

```
Version: DEV
Compiler: GNU (15.2.0)
IO Modes: packet_mmap_raw (default), packet_mmap, raw
```

JSON configuration:
```json
{
}
```

Steps to reproduce the behavior:
1. Follow the official install instructions
git clone https://github.com/rtbrick/bngblaster.git
cd bngblaster
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
make all
**Expected behavior**
bngblaster should install successfully

**Screenshots**


**Additional context**

need to modify variable in bngblaster/code/bngblaster/src/isis/isis_utils.c

--- a/code/bngblaster/src/isis/isis_utils.c
+++ b/code/bngblaster/src/isis/isis_utils.c
@@ -95,7 +95,7 @@ isis_str_to_area(const char *str, isis_area_s *area)
 {
     int len = 0;
     uint16_t *a;
-    char *ptr;
+    const char *ptr;
 
     if(!area->value) {
         area->value = calloc(1, ISIS_MAX_AREA_LEN);

